### PR TITLE
Add search box

### PR DESCRIPTION
**Pull request by lawong-d2l**
- Adding searchbox input with VUI styles
- Adding Clear Search button
- Updating class list api calls for searching
- Bumping to version 1.0.8

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -58,7 +66,8 @@ export default {
   name: 'activity-exemptions',
   data() {
     return {
-      locale: 'en'
+      locale: 'en',
```
Comment (jwong-d2l) [inline src/App.vue]: Does this need to be populated by the user's current locale, or is that part automagic?

File: src/App.vue [hunk 2]
```diff
@@ -238,7 +256,8 @@ tbody > tr:last-child td:last-child {
 }
 
 .d2l-button {
-  margin: 10px;
+  margin-top: 10px;
+  margin-right: 20px;
```
Comment (jwong-d2l) [inline src/App.vue]: Do you need the rtl version of this?

Comment (lawong-d2l) [reply src/App.vue]: Great catch. Fixed

File: src/App.vue (hunk 1 above)
Comment (lawong-d2l) [reply src/App.vue]: Out of scope for this guy, Ryan is looking into this one,

**Comment by jwong-d2l** [comment]
you've got conflicts!

File: package.json [hunk 3]
```diff
@@ -34,6 +35,8 @@
     "karma-phantomjs-launcher": "^1.0.4",
     "karma-webpack": "^2.0.4",
     "moxios": "^0.4.0",
+    "node-sass": "^4.5.3",
```
[the remark this answers is not in the document]
Comment (scasagrande-d2l) [reply package.json]: Less `node-sass` is `best-sass`

**Comment by RyanSweetD2L** [comment]
:shipit:

**Pull request merged**